requirements-docs.txt

```diff
@@ -1,5 +1,10 @@
 docutils<0.17,>=0.15  # sphinx-book-theme 0.2.0 has requirement docutils<0.17,>=0.15
 Sphinx<5,>=3  # sphinx-book-theme 0.3.3 has requirement sphinx<5,>=3
```

Comment:
Let's unpin this.

```suggestion
Sphinx
```

Reply:
@stevepiercy Let's sort this out separately. I was just trying to unblock merging the actual change here.